### DISCUSSION TITLE
Noam Brown: o1 was trained to do just do it all in one shot.

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/Raspberry#2] [Research]
![image](https://github.com/user-attachments/assets/a5b34cba-2a83-4ec1-b5d0-2e2d52492094)

> I wouldn't call o1 a "system". It's a model, but unlike previous models, it's trained to generate a very long chain of thought before returning a final answer

**Comment by daveshap** [comment]
This will keep the training data very simple and straight forward. Meaning we can do it with just one model/dataset. This is an open-and-shut finetuning job.

**Comment by russellballestrini** [comment]
https://youtu.be/KKF7kL0pGc4?si=HHQXbQ0un1sj2uUn
